### PR TITLE
HW3

### PR DESCRIPTION
**Pull request by julianoh7**
Hi Ahsan,
Last two questions took a very long time and can't solve it. 
Thank you.

### CODE REVIEW

**Comment by julianoh7** [comment]
check now

File: src/test/java/SeleniumHW3.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        String lowTempName = "low-temp-text";
+        By lowTemp = By.className(lowTempName);
+        WebElement lowBox = driver.findElement(lowTemp);
+
+        String highTempName = "high-temp-text";
+        By highTemp = By.className(highTempName);
+        WebElement highBox = driver.findElement(highTemp);
+
+        //feels-like temp >=low-temp && feels-like temp <= high temp
+
```
Comment (ahsansiddiki) [inline src/test/java/SeleniumHW3.java:177]: You can assertion like below
         boolean isTempWithinRange = false;
         if (feels >= low & feels<=high) {
         isTempWithinRange = true;
         }

         Assert.assertTrue(isTempWithinRange,"Feels like temperature is out of range.");

**Review by ahsansiddiki — COMMENTED** [review]
I added a comment otherwise looks great.

File: src/test/java/Locator2class.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        System.setProperty("webdriver.chrome.driver", "Drivers/chromedriver");
+        WebDriver driver = new ChromeDriver();
+        String url = "https://www.facebook.com/";
+        driver.get(url);
+        try {
+            Thread.sleep(5000);
+        }catch (InterruptedException e ){
+            e.printStackTrace();
+        }
+
```
[the remark this answers is not in the document]
Comment (ahsansiddiki) [reply src/test/java/Locator2class.java:18]: Actually this is a old comment. Don't be confused

**Comment by julianoh7** [comment]
Hi Ahsan,
If statement didn’t work am I doing something wrong?
Or do I get the value of feels, low, high temp and store in int variable to
do the equation?
Thanks.

**Comment by ahsansiddiki** [comment]
Look how someone else doing it

WebElement feelsLikeT = driver.findElement(By.className("feels-like-text"));
WebElement lowT = driver.findElement(By.className("low-temp-text"));
WebElement highT = driver.findElement(By.className("high-temp-text"));
String feelsLike = feelsLikeT.getText().substring(0,2); //removing degree
symbol
int feels = Integer.parseInt(feelsLike); // int from string
String lowTemp = lowT.getText().substring(0,2); //removing degree symbol
int low = Integer.parseInt(lowTemp);
String highTemp = highT.getText().substring(0,2); //removing degree symbol
int high = Integer.parseInt(highTemp);
boolean isTempWithinRange = false;
if (feels >= low & feels<=high) {
isTempWithinRange = true;
}
Assert.assertTrue(isTempWithinRange,"Feels like temperature is out of range.
");


Warm Regards

Ahsan Siddiki
(929)300-6788

**Pull request merged**